spec/models/user_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 describe User do
 
-  let(:therapist) { FactoryGirl.create(:therapist_user) }
-  let(:donor) { FactoryGirl.create(:donor_user) }
+  let(:therapist) { FactoryGirl.build(:therapist_user) }
```

Comment:
Figured we didn't need to actually commit the records to db since we're not using their `id` attributes in the tests